Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:latest
+WORKDIR /app
+
+RUN apt-get update -y && \
+    apt-get upgrade -y && \
+    apt-get install -y golang-go nodejs npm && \
+    npm install -g corepack && \
+    corepack enable && \
+    yarn install && \
+    apt-get install -y libgtk2.0-0t64 libgtk-3-0t64 libgbm-dev libnotify-dev libnss3 libxss1 libasound2t64 libxtst6 xauth xvfb && \
+    export DISPLAY=:0 && \
+    service dbus start
+
+RUN git clone https://github.com/scripthaus-dev/scripthaus.git
```

Comment:
Someone probably already has commented on this ...

But is this the github project you really intended to merge? note scripthaus-dev ...